abc.templates/infra/main.tf

```diff
@@ -0,0 +1,23 @@
+locals {
+  project_id                        = "REPLACE_PROJECT_ID"
+  automation_service_account_member = "REPLACE_AUTOMATION_SERVICE_ACCOUNT_MEMBER"
+}
+
+module "github_token_minter" {
+  source = "git::https://github.com/abcxyz/github-token-minter.git//terraform?ref=f1144811e6f6583b9aa971189ca8f6ef0f947c8a"
+
+  project_id = local.project_id
+
+  domains    = ["REPLACE_DOMAINS"]
```

Comment:
Just a note, as part of documentation, we should specify that a domain name will be required